docs/file-reference.md

```diff
@@ -155,6 +156,17 @@ name: mariadb
 | string   | yes          | The name of the app or micro-service this particular file defines. |
 
 
+## appversion
+
+```yaml
+appversion: "5.0.0-alpha"
```

Comment:
honestly, i'd put something like 0.0.1 instead of putting it in quotes. looks better / much easier.

Reply:
@cdrage I put it in quotes because, if someone is giving version as 5 or 5.1 then the user needs to give in quotes, it will work without quotes only in case of a string.
So I don't know what I should finally take as an example.

Reply:
I would actually prefer putting it in quotes.

Reply:
Let's use quotes for now, and than we can change is based on how #407 will be implemented

Reply:
I think it's ugly to have quotes... The power of YAML is that it can be either one when unmarshaling.

Based on when #455 is merged, we can make it whatever we want it to be (int or string) if someone were to put a float or a version value.

Reply:
We can't merge this before #407 anyway, so let's wait for how it will be implemented.

Reply:
@cdrage check https://github.com/kedgeproject/kedge/pull/454#discussion_r152602457